octopusdeploy_framework/schemas/channel.go

```diff
@@ -42,24 +42,18 @@ func (c ChannelSchema) GetResourceSchema() resourceSchema.Schema {
 				Description: "The project ID associated with this channel.",
 				Required:    true,
 			},
-			"rule": resourceSchema.ListNestedAttribute{
-				Description: "A list of rules associated with this channel.",
+			"space_id": GetSpaceIdResourceSchema(ChannelResourceDescription),
+			"tenant_tags": resourceSchema.ListAttribute{
```

Comment:
Should this have also been made a Set when migrating from Plugin => Framework? I'm not sure about the API behaviour here, are the ordered?